### ISSUE TITLE
Initiative: Theme development should not be necessary to use Backdrop CMS

### ISSUE THREAD

**Issue by cellear** [backdrop/backdrop-issues#4436]
Copied from: https://forum.backdropcms.org/forum/official-initiative-theme-development-should-not-be-necessary-use-backdrop-cms

Goal: People should be able to use backdrop without needing to know how to create or modify a theme.

Problem we're trying to solve:
Many potential users of Backdrop are not themselves skilled visual designers, and don’t have access to (or the resources to afford) a suitable designer. We should offer site builders the capability to create usable sites even if they lack either the technical ability to edit the HTML, PHP, CSS, JavaScript, and media elements that comprise most themes, or the design skills necessary to select visually-pleasing options.  Or both!  Backdrop is a content management system, we shouldn't require our users to bring to the table anything besides the content they are looking to manage.

Proposed MVP:
Find or construct three or four ready-to-go themes, and either include them with the basic download, or make them a one-click install.  

In order to be usable by the most inexperienced user, the themes should be:

- Usable out of the box, without requiring any customization, programming, or configuration.
- Fully tested and debugged, so that the can be expected to work for a common set of defined use cases
- Limited in number, so that anybody needed to support Backdrop will know how many ready-to-go themes are available and what they are
- Fully documented, so that users know what to expect without having to explore the themes themselves.

References:

- Wordpress's "Twenty-Something" themes.
- Wordpress.com's new site wizard
- Wix.com's new site wizard
- CSS Zen Garden
- Themeforest
- Drupal.org's occasional ready-to-go themes
- Drupal 8's Out-of-the-box initiative
- Drupal distributions like Commerce Toolkit and Open Atrium

**Comment by cellear** [comment]
I desperately want a shorter way to refer to this than "Initiative: Theme development should not be necessary to use Backdrop CMS"

Something like...

- Initiative 01
- No-Theme
- Ready-to-go
- Ready-to-wear
- OI #1

**Comment by docwilmot** [comment]
I think what this is about is just have more good themes in Backdrop contrib right? How about a title that says just that? The original post title and the suggestions above obscure that purpose to me. Lets just say "Better contrib themes", or "Contrib theme quality improvement initiative" or such.

**Comment by olafgrabienski** [comment]
> I think what this is about is just have more good themes in Backdrop contrib right? How about a title that says just that?

Good idea, sounds good to me (apart from "in contrib", could also be core). From https://forum.backdropcms.org/forum/official-initiative-theme-development-should-not-be-necessary-use-backdrop-cms: 

> For an initial MVP, I propose that we find or construct three or four ready-to-go themes, and either include them with the basic download, or make them a one-click install.

**Comment by oadaeh** [comment]
Based on my understanding of this initiative, I think @docwilmot's comment is more correct than the stated goal.

> Goal: People should be able to use backdrop without needing to know how to create or modify a theme.
Technically, that is possible now. One can install Backdrop and use Basis, along with the ability to do some minor customizations in the UI, and go about their day without needing to know anything about themeing.

However, I think doc's suggested alternatives are too broad and do not lend themselves to a SMART goal.

I have wanted to help with this initiative, but I have been waiting until I felt it was fleshed out and there were some clearly defined tasks to achieve the stated goal. I'm not sure if it has been fleshed out enough yet or not, but I think I can still help a little by using the Proposed MVP as a basis for activity:
> Find or construct three or four ready-to-go themes, and either include them with the basic download, or make them a one-click install.
Along with the 4 bullet points on what a theme should be to be considered for this.

@stpaultim created a spreadsheet some time ago with a list of the available themes at that time with some details and information about each theme:
https://docs.google.com/spreadsheets/d/15Cmp7jRsA_rpNXYPKsE1fNRGwjlu-m7rlKzIoirRJJI/edit?pli=1#gid=0

I'm going to update it and add some additional R2W-related data so that what is there could be used in consideration for this initiative.

**Comment by oadaeh** [comment]
Something I think would be good to help with selecting and installing relevant themes is to add some attribute to all themes that indicates their type or primary purpose, i.e., a base theme to build other themes from, an admin theme, etc., and maybe also a filter on that attribute. That way, people could easily hide options that are inappropriate for what they are looking for.

**Comment by stpaultim** [comment]
> Find or construct three or four ready-to-go themes, and either include them with the basic download, or make them a one-click install.

This is the currently stated MVP goal. It seems to me that we already have an easy way of installing themes through the UI of a Backdrop site. We also have a number of good contrib themes. What we don't have is any standard or criteria for telling whether or not a specific theme is "Ready to Wear."

At one point, our goal was to create a demo site with demo content that could be used to both TEST and SHOWCASE approved themes. 

To that end, we created a special issue queue for this demo site: https://github.com/backdrop-ops/demo/issues 
and have a demo site to experiment with: http://ready2wear.simplo.site/

It seems to me, that the next step is configuring the demo site with the kind of content and layouts that we want to test and approve a theme. 
* https://github.com/backdrop-ops/demo/issues/2
* https://github.com/backdrop-ops/demo/issues/3

**Comment by BWPanda** [comment]
What's the status of this initiative?

**Comment by cellear** [comment]
> 

well, I recently released a site for which I didn’t technically create a custom theme — though I did inject a good chunk of custom CSS into it, so I’m not sure if that counts. 

https://dreamspeak.com/

**Comment by BWPanda** [comment]
So do we close this issue as complete now? It's been another 12 months with no further discussion...
